ios/NativeBridge.swift

```diff
@@ -73,7 +73,11 @@ public class NativeBridge: NSObject {
         if(ephemeralSession) {
             let _ = builder.useEphemeralSession()
         }
-        let _ = builder.parameters(additionalParameters)
+        if(useSFSafariViewController) {
+            let _ = builder.provider(WebAuthentication.safariProvider())
```

Comment:
Should we add support for the various presentation styles that `SFSafariViewController supports`? E.g. https://github.com/auth0/auth0-flutter/blob/main/auth0_flutter_platform_interface/lib/src/web-auth/safari_view_controller.dart#L4C6-L4C43

Reply:
Adding it should be easy, but is it a feature that is used or requested. I don't want to provide too much options that wouldn't be used, but being the iOS expert, if our Swift and Flutter SDK has uses for it, I can add it here as well.

Reply:
This is a feature that allows developers to have some degree of control over the look and feel of the Safari View Controller. The user experience has always been a top priority for developers when it comes to the login flow, to reduce friction as much as possible. This allows them to fine-tune it a bit, which really helps.